### PR TITLE
OSDOCS-1142: Adding release notes for descheduler for 4.6

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://issues.redhat.com/browse/OSDOCS-1142

Preview: https://osdocs-1142-rn--ocpdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-6-release-notes.html#ocp-4-6-nodes

### CODE REVIEW

**Comment by bergerhoffer** [comment]
The build is expected to fail, as the docs that this links to have not been merged yet.

**Review by damemi — COMMENTED** [review]
Looks good but do we have the `ExcludeOwnerKinds` param (for RemoveDuplicates) documented? That was added to the operator for 4.6 in https://github.com/openshift/cluster-kube-descheduler-operator/pull/120

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -377,6 +377,23 @@ and no other Reports or ReportQueries depend on the expiring Report, the Meterin
...
+The descheduler now allows you to configure the `PodLifeTime` strategy. This strategy evicts pods after they reach a certain, configurable age.
+
+See xref:../nodes/scheduling/nodes-descheduler.adoc#nodes-descheduler-strategies_nodes-descheduler[Descheduler strategies] for more information.
+
+[id="ocp-4-6-descheduler-filtering"]
+==== Descheduler filtering by namespace and priority (Technology Preview)
+
+You can now configure whether descheduler strategies should consider pods for eviction based on their namespace and priority.
+
+See xref:../nodes/scheduling/nodes-descheduler.adoc#nodes-descheduler-filtering-namespace_nodes-descheduler[Filtering pods by namespace] and xref:../nodes/scheduling/nodes-descheduler.adoc#nodes-descheduler-filtering-priority_nodes-descheduler[Filtering pods by priority] for more information.
```
Comment (bmcelvee) [inline release_notes/ocp-4-6-release-notes.adoc:395]: These two links are resolving to the top of the 
Evicting Pods using the descheduler page.

**Comment by bmcelvee** [comment]
Just a note about two of the links, otherwise LGTM! :dog:

**Comment by bmcelvee** [comment]
Sorry!! I hit the wrong button.

File: release_notes/ocp-4-6-release-notes.adoc (hunk 1 above)
Comment (bergerhoffer) [reply release_notes/ocp-4-6-release-notes.adoc:395]: @bmcelvee Yeah sorry, I mentioned that in an earlier comment in the PR, should have mentioned it in slack too. These won't go down to the proper location until #26224 is merged.

**Review by damemi — APPROVED** [review]
/lgtm

**Pull request merged**